tools/linker/MobileSweepStep.cs

```diff
@@ -72,5 +72,13 @@ protected override void InterfaceRemoved (TypeDefinition type, InterfaceImplemen
 				LinkContext.ProtocolImplementations [type] = list = new List<TypeDefinition> ();
 			list.Add (iface.InterfaceType.Resolve ());
 		}
+
+		protected override void ElementRemoved (IMetadataTokenProvider element)
```

Comment:
`Mobile*` should be shareable (not too far away I hope) with XA
This should be moved into `CoreSweepStep` so it can be re-used by other, mobile-like, profiles